test/lib/user_permissions_exporter_test.rb

```diff
@@ -1,82 +0,0 @@
-require "test_helper"
-
-class UserPermissionsExporterTest < ActionView::TestCase
-  def setup
-    @chips_org = create(:organisation, name: "Ministry of chips")
```

Comment:
Slightly sad to lose a reference to the "ministry of chips" 😂

Reply:
😢